packages/pagination/stories.tsx

```diff
@@ -51,17 +54,25 @@ storiesOf('Pagination Container', module)
       });
 
       return (
-        <nav>
+        <nav aria-label="Pagination (Hook)">
           <ul
             {...getContainerProps({
+              role: null,
```

Comment:
If we have to provide `role: null` in our usages why do we need to introduce it into the prop-getters?

Reply:
This is a pattern throughout `react-containers`. We express the correct `role` so that users can spread the props on to non-semantic markup. However, when we use semantic markup, it's correct to knock out the role as the first rule of ARIA is no ARIA when it's not needed.

Per the PR description, the container example below shows how the role can be applied to less smart (`<div>`) markup.